docs/src/scss/uswds-site.scss

```diff
@@ -1,6 +1,11 @@
 @use "uswds-core" as *;
 @use 'sass:math';
 
+// sass not compiling in uswds - temp fix
```

Comment:
I'd like to get a backlog ticket in to look into this further, and remove this eventually.

Reply:
#1299